Orange-OpenSource/Orange-Boosted-Bootstrap
# Prepare v4.5.3

Closes #485 

  - Firefox ESR
  - IE11
  - Latest Edge, Chrome, Firefox, Safari
  - iOS Safari
  - Chrome & Firefox on Android
  - `conventional-changelog -p angular -i CHANGELOG.md -s`
  - ship list: #485 
- [x] merge (on `v4-dev` for now)
  - attach zip file
  - paste CHANGELOG in the release's description
- [ ] run `build/ship.sh x.x.x` to…
  - Pull the latest changes update new version numbers
  - Compile CSS and JS
  - Compress dist files
  - Generate the hashes and update our config's cdn section
  - Compile the docs with `JEKYLL_ENV=production` (for v4)
  - Copy-pasta built docs to `bs-docs`-named clone of Boosted.
  - [ ] copy `../bs-docs` to the `gh-pages` branch
  - if you are already logged in NPM (with a personnal account, for example), [you'd better use a repository scoped `.npmrc` file](https://stackoverflow.com/questions/30114166/how-to-have-multiple-npm-users-set-up-locally)
  - if you're releasing a pre-release, use `--tag`, eg for v5-alpha1 `npm publish boosted-5.0.0-alpha1.tgz --tag next`
